src/PrestaShopBundle/Resources/config/routing/admin/configure/shop_parameters/contacts.yml

```diff
@@ -0,0 +1,46 @@
+admin_contacts:
+    path: /
+    methods: GET
+    defaults:
+        _controller: 'PrestaShopBundle:Admin\Configure\ShopParameters\Contacts:index'
+        _legacy_controller: AdminContacts
+
+admin_contacts_search:
+    path: /
+    methods: [POST]
+    defaults:
+        _controller: 'PrestaShopBundle:Admin\Configure\ShopParameters\Contacts:search'
+        _legacy_controller: AdminContacts
+
+
+admin_contacts_create:
+    path: /create
+    methods: GET
+    defaults:
+        _controller: 'PrestaShopBundle:Admin\Configure\ShopParameters\Contacts:create'
+        _legacy_controller: AdminContacts
+
+admin_contacts_edit:
+    path: /edit/{contactId}
+    methods: GET
+    defaults:
+        _controller: 'PrestaShopBundle:Admin\Configure\ShopParameters\Contacts:edit'
+        _legacy_controller: AdminContacts
+    requirements:
+        contactId: \d+
+
+admin_contacts_delete:
+    path: /delete/{contactId}
+    methods: [GET]
```

Comment:
better use POST

Reply:
I can change it to `DELETE` for single delete action, but bulk actions do not allow `DELETE` method so it would be inconsistent :no_mouth: 
https://github.com/PrestaShop/PrestaShop/blob/develop/src/Core/Grid/Action/Bulk/Type/SubmitBulkAction.php#L60

Reply:
@mickaelandrieu can we extend SubmitBulkAction to allow DELETE HTTP requests or is there a technical issue with this ?